src/components/stopwatch/StopWatchController.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <FreeRTOS.h>
+#include <timers.h>
+
+#define LAP_CAPACITY 2
```

Comment:
This should be a constexpr constant

Reply:
Done, hopefully I placed it correctly (I'm not a C++ native).